actionpack/lib/action_controller/metal/streaming.rb

```diff
@@ -73,7 +73,7 @@ module ActionController #:nodoc:
   # or not, is by using +content_for+, +provide+ and +yield+.
   #
   # Take a simple example where the layout expects the template to tell
-  # which title to use:
+  # it which title to use:
```

Comment:
"Take a simple example where the layout expects the template to be told which title to use:" ?